LinqToRest.Test/QueryProviderTest.cs

```diff
@@ -20,7 +20,7 @@ public void ReturnsRestQuery()
 
             Assert.Equal(expected, actual);
         }
-
+       

```

Comment:
Remove whitespaces

Reply:
Addressed in https://github.com/messerli-informatik-ag/linq-to-rest/pull/27/commits/c6de22bdf8e765326f25d3e6fc03ce169e79b74a